src/emulator/storage/apis/gcloud.ts

```diff
@@ -43,19 +41,16 @@ export function createCloudEndpoints(emulator: StorageEmulator): Router {
     async (req, res) => {
       let getObjectResponse: GetObjectResponse;
       try {
-        getObjectResponse = await storageLayer.handleGetObject(
-          {
-            bucketId: req.params.bucketId,
-            decodedObjectId: req.params.objectId,
-          },
-          /* skipAuth = */ true
-        );
+        getObjectResponse = await adminStorageLayer.handleGetObject({
+          bucketId: req.params.bucketId,
+          decodedObjectId: req.params.objectId,
+        });
       } catch (err) {
         if (err instanceof NotFoundError) {
           return res.sendStatus(404);
         }
         if (err instanceof ForbiddenError) {
-          throw new Error("Request failed unexpectedly due to Firebase Rules.");
+          return res.sendStatus(403);
```

Comment:
Previously, the only way a `ForbiddenError` could be thrown is if Firebase Rules auth failed. This represented an error state since we didn't expect Firebase Rules to be run at all when executing admin operations.

Now, a `ForbiddenError` is an acceptable error state since the `StorageLayer` can deny a request if the admin credential validator fails, so we return a 403.